docs/collector.ohwm.md

```diff
@@ -0,0 +1,35 @@
+# ohwm collector
```

Comment:
```suggestion
# open_hardware_monitor collector
```

Just name it like that. By full name. People may better understand what it does and people google for it, they may find this. No one knows what ohwm could be unless they read the docs.

It would be great to have some small docs about how I can install the Open Hardware Monitor